### DISCUSSION TITLE
Deploy by Docker -> Can't load the image after uploaded.

### DISCUSSION THREAD

**Discussion by melodyxpot** [vercel/next.js#57065] [App Router]
Hi.
I am facing to the image uploading problem.
I set up the `volumes` to store the image on local from container.
```version: "3"
services:
  frontend:
    container_name: xipics_frontend
    build:
      context: .
    ports:
      - "3010:3000"
    restart: always
    volumes:
      # Set up the uploads file folder
      - ./public/uploads:/app/public/uploads
  nginx:
    container_name: xipics_nginx
    image: nginx:alpine
    restart: always
    ports:
      - "7009:80"
    volumes:
      - ./docker/nginx.conf:/etc/nginx/conf.d/default.conf
    depends_on:
      - frontend
```
When I restart the container, I can load the images I uploaded but can't immediately load after uploaded image.
How can I fix this problem?
I think this is also cache issue I faced in dynamic API routing.
Thanks for help in advance

**Comment by wwwdepot** [comment]
Read "Good to know" section in https://nextjs.org/docs/app/building-your-application/optimizing/static-assets
